components/host-dashboard/ScheduledExpensesBanner.js

```diff
@@ -0,0 +1,136 @@
+    context: API_V2_CONTEXT,
+  });
+  const { addToast } = useToasts();
+  const [showConfirmationModal, setConfirmationModalDisplay] = React.useState(false);
+
+  const hasScheduledExpenses = scheduledExpenses.data?.expenses?.totalCount > 0;
+  if (!hasScheduledExpenses) {
+    return null;
+  }
+
+  const request = create({ mode: process.env.WISE_ENVIRONMENT === 'production' ? Mode.PRODUCTION : Mode.SANDBOX });
+  const handlePayBatch = async () => {
+    const expenseIds = scheduledExpenses.data.expenses.nodes.map(e => e.id);
+    try {
+      await request(`${process.env.WEBSITE_URL}/api/services/transferwise/pay-batch`, {
+        method: 'POST',
+        body: JSON.stringify({ expenseIds, hostId: host.id }),
+        headers: addAuthTokenToHeader(),
+      });
+      setConfirmationModalDisplay(false);
+      addToast({
+        type: TOAST_TYPE.SUCCESS,
+        message: (
+          <FormattedMessage
+            id="expenses.scheduled.paybatch.success"
+            defaultMessage="Expenses paid! They're now being processed by Wise."
+          />
+        ),
+      });
+      onSubmit?.();
+    } catch (e) {
+      addToast({ type: TOAST_TYPE.ERROR, message: e.message });
+    }
+  };
+
+  return (
+    <React.Fragment>
+      <MessageBox type="success" mb={4}>
+        <Flex alignItems="center" justifyContent="space-between">
+          <Box>
+            <TransferwiseIcon size="1em" color="#25B869" mr={2} />
+            <FormattedMessage
+              id="expenses.scheduled.notification"
+              defaultMessage="You have {count, plural, one {# expense} other {# expenses}} scheduled for payment."
+              values={{ count: scheduledExpenses.data.expenses.totalCount }}
+            />
+          </Box>
+          <Box>
+            {secondButton}
+            <StyledButton
+              buttonSize="tiny"
+              buttonStyle="successSecondary"
+              onClick={() => setConfirmationModalDisplay(true)}
+            >
+              <FormattedMessage id="expenses.scheduled.paybatch" defaultMessage="Pay Batch" />
+            </StyledButton>
+          </Box>
+        </Flex>
+      </MessageBox>
+      <ConfirmationModal
+        show={showConfirmationModal}
+        header={<FormattedMessage id="expenses.scheduled.confirmation.title" defaultMessage="Pay Expenses Batch" />}
+        body={
+          <FormattedMessage
+            id="expenses.scheduled.confirmation.body"
+            defaultMessage="Are you sure you want to batch and pay {count, plural, one {# expense} other {# expenses}} scheduled for payment?"
+            values={{ count: scheduledExpenses.data.expenses.totalCount }}
+          />
+        }
+        onClose={() => setConfirmationModalDisplay(false)}
+        cancelLabel={<FormattedMessage id="actions.cancel" defaultMessage="Cancel" />}
+        cancelHandler={() => setConfirmationModalDisplay(false)}
```

Comment:
:ice_cream: This one defaults to `onClose`. We should simplify the props for this component at some point.

```suggestion
```